### ISSUE TITLE
ifcopenshell Settings cause issues creating Topology object ByIFCFile.

### ISSUE THREAD

**Issue by benjaminwfriedman** [wassimj/topologicpy#65]
When attempting to create a topology object from an IFC file, I hit the following error: 
```
Traceback (most recent call last):
  File "D:\Repos\TopologicTestCases\main.py", line 3, in <module>
    building = Topology.ByIFCFile("D:\Repos\TopologicTestCases\Autodesk-Research_210-King.ifc")
  File "D:\Repos\TopologicTestCases\venv\lib\site-packages\topologicpy\Topology.py", line 2190, in ByIFCFile
    settings.set(settings.DISABLE_TRIANGULATION, True)
  File "D:\Repos\TopologicTestCases\venv\lib\site-packages\ifcopenshell\geom\main.py", line 212, in __getattr__
    raise AttributeError("'Settings' object has no attribute '%s'" % k)
AttributeError: 'Settings' object has no attribute 'DISABLE_TRIANGULATION'
```
This looks to emanating from an issue trying access a ifcopenshell setting that is not available. 

Seems similar to a fix that was needed in commit [2fbc86d](https://github.com/wassimj/topologicpy/commit/2fbc86db55fef73fb8f226138bb178b7287aba85)

using topologicpy v 0.7.40

**Comment by wassimj** [comment]
Thanks @benjaminwfriedman,
I was dealing with this bug as you published this issue :)  The problem was that ifcopenshell changed their API and the changes are not well-documented. What is worse, the deprecated code is still part of their documentation. Nevertheless, I sought help and found out the new code. I believe this is now resolved in v0.7.41. Please test and if you are satisfied, please close this issue. Thanks!

**Comment by benjaminwfriedman** [comment]
Excellent! I was getting started on a PR for it haha. I will give it a test!
